BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -380,21 +380,21 @@ Implementers should note that compression at higher protocol levels can allow an
 ## TLS Session Resumption
 {: #rec-resume}
 
-Session resumption drastically reduces the number of TLS handshakes and thus is an essential
+Session resumption drastically reduces the number of full TLS handshakes and thus is an essential
 performance feature for most deployments.
 
 Stateless session resumption with session tickets is a popular strategy. For TLS 1.2, it is specified in
 {{?RFC5077}}.  For TLS 1.3, a more secure PSK-based mechanism is described in
-{{Section 4.6.1 of RFC8446}}. See {{Springall16}} for a quantitative study of TLS cryptographic "shortcuts", including session resumption.
+{{Section 4.6.1 of RFC8446}}. See {{Springall16}} for a quantitative study of the risks induced by TLS cryptographic "shortcuts", including session resumption.
 
 When it is used, the resumption information MUST
 be authenticated and encrypted to prevent modification or eavesdropping by an attacker.
 Further recommendations apply to session tickets:
 
-* A strong cipher suite MUST be used when encrypting the ticket (as least as strong as the main TLS cipher suite).
-* Ticket keys MUST be changed regularly, e.g., once every week, so as not to negate the benefits of forward secrecy (see {{sec-pfs}} for details on forward secrecy). Old ticket keys MUST be destroyed shortly after a new key version is made available.
-* For similar reasons, session ticket validity MUST be limited to a reasonable duration (e.g., half as long as ticket key validity).
-* TLS 1.2 does not roll the session key forward within a single session. Thus, to prevent an attack where a stolen ticket key is used to decrypt the entire content of a session (negating the concept of forward secrecy), a TLS 1.2 server SHOULD NOT resume sessions that are too old, e.g. sessions that have been open longer than two ticket key rotation periods. Note that this implies that some server implementations might need to abort sessions after a certain duration.
+* A strong cipher MUST be used when encrypting the ticket (as least as strong as the main TLS cipher suite).
```

Comment:
In a TLS (1.2) context a "cipher suite" is the combination of symmetric cipher, key-exchange mechansim, server authentication, and MAC.  Encrypting session tickets requires only symmetric cipher and MAC, which in the AEAD abstraction is just a "cipher" rather than a "cipher suite".